docs/OrganizingCommittee/Committee-formation.md

```diff
@@ -0,0 +1,84 @@
+
+## overall coordination
+
+There may be one or several people considered overall leaders of the event. These are often the people who initiated the idea of the hack week in the first place. Within a more traditional organizational structure, there may be one person responsible for most of the decision making and ensuring everthing gets done, however this can place a significant burden on a single person. In a more distributed leadership model, multiple people have autonomous decision making power informed and supported by cross-team communications. 
+
+Responsibilities of overall coordinators include:
+
+* providing the impetus and energy for initiating and moving the event forward
+* calling meetings, establishing effective channels of communication and linking up the right people
+* identifying duplication of effort and increasing team efficiencies
+* acting as the main point of contact for outreach and communcations outside of the group
+* writing proposals and seeking funding
+* leading efforts to recruit members of the organizing committee
+* keeping an eye on the big picture to be sure everyone is on schedule
+* leading challenging discussions and mediating conflicts
+
+# Recruiting an organizing committee
+
+Depending on the size of the group who initiate the idea of a hack week, it may or may not be necessary to put time into recruiting others to assist as members of the organizing committee. In the case of an institution such as a university department starting a hack week, there may be enough local administrative support and tutorial leads without needing to find additional people. In other cases, for example our recent ICESat-2 hack week aimed at building software tools for a new NASA mission, we wanted to have participation from a broad community, especially members of the satellite's science team who had already begun to build software tools. In this case a broader range of recruitment efforts were necessary.
+
+Because the organizing committee has a responsibility to model the principles and norms of behavior we aim to promote at a hack week, it is important to find people whose values align with this mission. Similarly, we want the organizing committee to be as diverse as possible to maximize the number of different perspectives in the room, and to offer role models to an equally diverse range of hack week participants.  
+
+Here are some of general principles for recruiting a diverse and engaged organizing committee:
+
+* Designate one person or a small committee to be responsible for recruitment. Among this group, build a list of potential recruitees from each person's network of trusted colleagues. This list may also contain names of people who have approached the hack week organizers with an intention to participate. This is common for hack weeks that span multiple years, where past participants wish to be more engaged in subsequent events. 

```

Comment:
Is it worth pointing out that the group should think deeply about who in their network might be good organizers, and not stop with the first 2-3 suggestions? In my experience (and I think there's research on that, too), the first few candidates that spring to mind will be white and male, which might mean that you don't end up with a very diverse committee. It's also worth thinking about whether the first couple of candidates that come to mind beyond that are minority people in the network who are already overburdened with organizational tasks they're asked to do. One suggestion could be to figure out whether there are particular people in the organizers' networks who would be great organizers and also benefit from being on the committee.